packages/components/src/components/date-picker/date-picker.scss

```diff
@@ -119,15 +115,14 @@
             &-text {
                 background-color: inherit;
                 padding-top: 0.2rem;
-                color: var(--text-prominent);
 
                 &--has-error {
                     padding-top: unset;
-                    color: var(--brand-red-coral);
+                    color: var(--brand-red-coral) !important;
```

Comment:
Any reason why `!important` is needed since the issue is only reported on mobile?

Reply:
The properties from `text` component are overriding this CSS. I can add extra logic to handle all this inside our component as well, but I just thought it's a more elegant solution to keep CSS modifiers, and don't put extra lines of logic. Maybe I can just remove color property from the component and let CSS handle it.